keps/2076-kueuectl/kep.yaml

```
@@ -0,0 +1,28 @@
+title: Kueuectl
+kep-number: 2076
+authors:
+  - "@mwielgus"
+status: implementable
+creation-date: 2024-04-25
+reviewers:
+  - "@alculquicondor"
+approvers:
+  - "@alculquicondor"
+replaces:
+  - "/keps/487-kubectl-plugin"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v0.8"
```

```suggestion
latest-milestone: "v0.7"
```

@mwielgus Isn't this target v0.7? Because we already implemented the part of the command.

---

Although we will likely not finish a meaningful amount of commands. Not sure.

---

If your team doesn't have enough time to implement commands, I'm ok with postponing it to v0.8.

---

/lgtm